htdocs/web_portal/GOCDB_monitor/tests.php

```diff
@@ -134,7 +134,13 @@ function test_db_connection()
     $retval = [];
     try {
         $entityManager = Factory::getNewEntityManager();
-        $entityManager->getConnection()->connect();
```

Comment:
Is `connect` not needed?

Reply:
Does it require `;` to close off the SQL statement?

Reply:
It does not 😄 https://www.doctrine-project.org/projects/doctrine-dbal/en/3.0/reference/data-retrieval-and-manipulation.html#fetchone